pulsenotify/main.py

```diff
@@ -24,4 +23,11 @@ def cli():
 
 
 if __name__ == '__main__':
+    # Get desired log level from environment, fallback to "INFO"
```

Comment:
I don't think this comment gives more context to the line below. I believe we can live without it